File: o/soft_robot/derived/ikko_dake/eqs/julia_style/K.jl
module G
function f(q::Vector{T}, q_dot::Vector{T}, xi::T) where T
    l1 = q[1]
    l2 = q[2]
    l3 = q[3]
    l1_dot = q_dot[1]
    l2_dot = q_dot[2]
    l3_dot = q_dot[3]
    
    [1700    0    0;
0 1700    0;
0    0 1700]
end
end

File: o/soft_robot/derivation_of_dynamics/derived/ikko_dake/eqs/numpy_style/K.py
import numpy
def f(q, q_dot, xi):
    l1, l2, l3 = q[0,0], q[1,0], q[2,0]
    l1_dot, l2_dot, l3_dot = q_dot[0,0], q_dot[1,0], q_dot[2,0]

    return numpy.array([[1700, 0, 0], [0, 1700, 0], [0, 0, 1700]])